packages/interactivity/docs/2-api-reference.md

```diff
@@ -449,6 +450,32 @@ store( {
 
 The `wp-init` can return a function. If it does, the returned function will run when the element is removed from the DOM.
 
+#### `wp-key` 
+
+Here is the text for the `wp-key` directive description:
+
+The `wp-key` directive assigns a unique key to an element to help the Interactivity API identify it when iterating through arrays of elements. This becomes important if your array elements can move (e.g. due to sorting), get inserted, or get deleted. A well-chosen key value helps the Interactivity API infer what exactly has changed in the array, allowing it to make the correct updates to the DOM.
```

Comment:
For now, I think it's useful only when doing client-side navigation. So maybe we can add that in https://github.com/WordPress/gutenberg/pull/53733.

Ok, that makes sense. If you don't mind, let's merge this as it is because it's merged against https://github.com/WordPress/gutenberg/pull/53733, and let's add a better example of `data-wp-key` in the docs for that PR.